components/webfield/AreaChairConsole.js

```diff
@@ -122,10 +122,14 @@ const AssignedPaperRow = ({
 }
 
 const AreaChairConsoleTasks = ({ venueId, areaChairName }) => {
+  const areaChairUrlFormat = areaChairName ? getSingularRoleName(areaChairName)
```

Comment:
the changes look good to me, I wonder if this code can be moved to an util function, it seems to be duplicated in different places.

Reply:
Fixed, I called it `getRoleHashFragment` but maybe the name can be improved.